lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -518,9 +518,13 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
             final Map<String, dynamic> arguments = decoded.arguments as Map<String, dynamic>;
             // TODO(ferhat): Find more appropriate defaults? Or noop when values are null?
             final String label = arguments['label'] as String? ?? '';
-            final int primaryColor = arguments['primaryColor'] as int? ?? 0xFF000000;
```

Comment:
I think it's fair that we drop this color from here. The API docs say:

> Specifies the description of the current state of the application as it pertains to the application switcher (also known as "recent tasks").
> 
> Any part of the description that is unsupported on the current platform will be ignored.

Web doesn't support the recent tasks at all. It's OK that we set the title of the page, but the color is not needed here. The new location is way more appropriate.